openapi-specs/cspm/CommandCenterMicroService.json

```diff
@@ -478,7 +477,7 @@
     "/v1/vulnerabilities/summary": {
       "post": {
         "tags": [
-          "Command Center"
+          "Command Center Swimlane APIs"
```

Comment:
Please retain the tag name as Command center

Reply:
Restored in latest commit